### DISCUSSION TITLE
Viral load analysis in host organism cells

### DISCUSSION THREAD

**Discussion by tkapello** [satijalab/seurat#9293] [General]
Hi all,

I was wondering whether there is a consensus about how to test viral infection of cell types in a scRNA-seq dataset. I have mapped my reads to viral genomes and I have a matrix of viral genes x cell barcodes. I want to investigate which clusters are infected but I am not sure whether to use the `nCount_RNA` or `nFeature_RNA`. Is there a standard pipeline in the field?

**Comment by samuel-marsh** [comment]
Hi,

Not member of dev team but hopefully can be helpful.  As this is not a question about the Seurat package itself I'm going to move this to discussion section and hopefully other members of community working on similar projects can chime in.

Best,
Sam
